src/components/parameter.jsx

```diff
@@ -114,14 +120,12 @@ const Parameter = React.createClass({
           <div className="page-header">
             <h1>{descriptionMessage}</h1>
           </div>
-          {this.renderDefinitionsList(parameter)}
-          <hr/>
           <div className="row">
             <div className="col-lg-8">
               {
-                type === "Parameter"
-                  ? this.renderParameter(parameter)
-                  : this.renderScale(parameter)
+                type === "Scale"
```

Comment:
I don't think that this test is relevant anymore with the new API, which removes the `parameter['@type']` property (cf line above `const type = parameter["@type"]`)